ci/tests/replica.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+#imports
+. ci/libs/variables.sh
+. ci/libs/utils.sh
+
+test_single_replica_stop_start() {
+	echo "----------------Test_single_replica_stop_start--------------"
+	orig_controller_id=$(start_controller "$CONTROLLER_IP" "store1" "1")
```

Comment:
```suggestion
	orig_controller_id=$(start_controller "${CONTROLLER_IP}" "store1" "1")
```

Reply:
@farhaanbukhsh 
What's the point of doing this?
Both are same. Right?

Reply:
Same question from my side also